prog/weaver/main.go

```diff
@@ -106,6 +107,7 @@ func main() {
 	mflag.StringVar(&dbPrefix, []string{"-db-prefix"}, "weave", "pathname/prefix of filename to store data")
 
 	mflag.StringVar(&trustedSubnetStr, []string{"-trusted-subnets"}, "", "Comma-separated list of trusted subnets in CIDR notation")
+	mflag.StringVar(&ipamSeedStr, []string{"-ipam-seed"}, "", "Comma-separated list of peer names used to form initial ring")
```

Comment:
nothing anywhere in our docs mentions "ring". and rightly so. We need to explain this in terms that don't delve into implementation detail.